artview/components/menu.py

```diff
@@ -264,13 +268,13 @@ def _about(self):
         txOut = "This is a simple radar file browser to allow \
                  quicklooks using the DoE PyArt software"
         QtGui.QMessageBox.about(self, "About ARTView", txOut)
- 
+
     def _get_RadarLongInfo(self):
-        '''Print out the radar info to text box.'''
+        '''Print out the radar info to text box and terminal.'''
```

Comment:
I forgot that I had not found a way to print this to a text window yet.
Just made a note to look into this.